migration/v0.27.x-v0.28.x/upgrade-apps.md

```diff
@@ -28,10 +28,6 @@
     ./migration/v0.27.x-v0.28.x/add-registry-credentials.sh
     ```
 
-1. Run migration script `remove_old_metrics_server.sh`
-
-    This will remove the old metrics-server components from the clusters.
-
```

Comment:
was this added to the wrong document?

Reply:
I assume so, as the chart upgrade is not part of the release and this script is in `[v0.28.x-v0.29.x](https://github.com/elastisys/compliantkubernetes-apps/tree/main/migration/v0.28.x-v0.29.x)` migration folder